mcodingbot/plugins/peps.py

```diff
@@ -89,19 +127,33 @@ def within_age_cutoff(message_created_at: datetime) -> bool:
     return datetime.now(timezone.utc) - message_created_at <= MAX_AGE_FOR_SEND
 
 
-def get_peps_embed(content: str) -> hikari.Embed | None:
-    pep_refs = [
-        int(ref.group("pep")) for ref in re.finditer(PEP_REGEX, content)
-    ]
-    peps = map(pep_manager.get, sorted(set(pep_refs))[:5])
-    pep_links_message = "\n".join(str(pep) for pep in peps if pep)
+def get_pep_refs(content: hikari.UndefinedNoneOr[str]) -> list[PEPInfo]:
+    """
+    Return a sorted list of all the peps mentioned in a string.
+    """
+    if not content:
+        return []
+
+    peps = sorted(
+        [int(ref.group("pep")) for ref in re.finditer(PEP_REGEX, content)]
+    )
+
+    return [pep for pep in map(pep_manager.get, peps) if pep]
```

Comment:
Maybe?
```suggestion
    return list(filter(None, map(pep_manager.get, peps)))
```

Reply:
imo the original is better

Reply:
I prefer tragic's suggestion for style because it doesn't mix iterator functions and list comps. List comp more readable tho cause tbh I didn't even know this was a python feature until now so I'm not sure.

Reply:
The only reason I prefer the original is due to readability.

Reply:
I change my mind. I like trag1c